resources/views/meetups/edit.blade.php

```diff
@@ -1,43 +1,107 @@
+                                class="w-full sm:w-[400px] h-[200px]">
+                            <input type="file" accept="image/*" id="image-input" class="hidden">
+                            <label for="image-input"
+                                class="inline-flex items-center px-4 py-2 border border-transparent shadow-sm text-base font-medium rounded-md text-white bg-indigo-600 hover:bg-indigo-700 focus:outline-none focus:ring-2 focus:ring-offset-2 focus:ring-indigo-500 cursor-pointer mx-auto w-full justify-center mt-3 sm:w-auto sm:mt-0">
+                                <!-- Heroicon name: solid/mail -->
+                                <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6 mr-2" fill="none" viewBox="0 0 24 24"
+                                    stroke="currentColor" stroke-width="2">
+                                    <path stroke-linecap="round" stroke-linejoin="round"
+                                        d="M4 16v1a3 3 0 003 3h10a3 3 0 003-3v-1m-4-8l-4-4m0 0L8 8m4-4v12" />
+                                </svg>
+                                Upload Image
+                            </label>
+                        </div>
+                        <div>
+                            <label for="title" class="block font-medium text-gray-700">
+                                Title
+                            </label>
+                            <div class="mt-1">
+                                <input type="text" name="title" id="title" placeholder="Title..."
+                                    value="{{ old('title', $meetup->title) }}"
+                                    class="shadow-sm focus:ring-indigo-500 focus:border-indigo-500 block w-full sm:text-sm border-gray-300 rounded-md" />
+                                <x-input-error for="title" />
+                            </div>
+                        </div>
+
+                        <div>
+                            <label for="description" class="block font-medium text-gray-700">
+                                Description
+                            </label>
+                            <div class="mt-1">
+                                <textarea rows="5" type="text" name="description" id="description" placeholder="Description..."
+                                    class="shadow-sm focus:ring-indigo-500 focus:border-indigo-500 block w-full sm:text-sm border-gray-300 rounded-md">{{ old('description', $meetup->description) }}</textarea>
+                                <x-input-error for="description" />
+                            </div>
+                        </div>
+
+                        <div>
+                            <label for="date" class="block font-medium text-gray-700">
+                                Date
+                            </label>
+                            <div class="mt-1">
+                                <input type="date" name="date" id="date" placeholder="Date..."
```

Comment:
```suggestion
                                <input type="datetime-local" name="date" id="date" placeholder="Date..."
```
Maybe show and input date and Time not only date